BParkingNano/python/tracksBPark_cff.py

```diff
@@ -6,13 +6,13 @@
                              trgMuon    = cms.InputTag("muonTrgSelector:trgMuons"),
                              tracks     = cms.InputTag("packedPFCandidates"),
                              lostTracks = cms.InputTag("lostTracks"),
-                             trkPtCut = cms.double(0.5),    
+                             trkPtCut = cms.double(0.7),
                              muons      = cms.InputTag("slimmedMuons"),
                              pfElectrons= cms.InputTag("slimmedElectrons"),
                              vertices   = cms.InputTag("offlineSlimmedPrimaryVertices"),
                              trkEtaCut = cms.double(2.5),
-                             dzTrg_cleaning = cms.double(-1.), ##next step change to 1
-                             drTrg_Cleaning = cms.double(-0.4),  ##next step move to 0.01
+                             dzTrg_cleaning = cms.double(1.),
+                             drTrg_Cleaning = cms.double(0.03),
```

Comment:
this will kill the tag muon. Are we sure that we do not need it even for x-check?

Reply:
do you mean it will reduce the signal on the tag side?
I don't have values in mind for dR, what I can say is that if we ask minDR > 0.03 (as it is now) for final particles in the B cand, this cut will not harm. Things are different if we decide to decrease minDR (as I think you are proposing, right?)